lib/alchemy/tasks/helpers.rb

```diff
@@ -11,7 +11,7 @@ def database_import_command(adapter)
       end
 
       def database_config
-        raise "Could not find #{database_config_file}!" if !File.exists?(database_config_file)
+        raise "Could not find #{database_config_file}!" unless File.exists?(database_config_file)
```

Comment:
Lint/DeprecatedClassMethods: `File.exists?` is deprecated in favor of `File.exist?`.